Pear/Settings/EditInterestsViewController.swift

```diff
@@ -25,6 +25,8 @@ class EditInterestsViewController: UIViewController {
     // MARK: - Private Data Vars
     private var sections: [EditSection<Interest>] = []
     private let user: UserV2
+    
+    var profileDelegate: ProfileMenuDelegate?
```

Comment:
private?

Reply:
Why any object?

Reply:
Weak can only be defined for reference types, so we have to conform the protocol must conform to a class. Before Swift for this meant writing `protocol name: class { }` but after Swift 4 the convention is `protocol name: AnyObject { }`